src/it/xerces/src/test/java/com/jcabi/xerces/XercesSampleTest.java

```diff
@@ -56,6 +57,10 @@
      */
     @Test
     public void validatesXmlForSchemaValidity() throws Exception {
+        final int zero = 0;
```

Comment:
@tmjee There is not a big sense declaring one-time variable (especially with such names). It's better to inline them all and put checkstyle suppression.